bin/slack-alert

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+ENVIRONMENT_NAME=$1
+BUILD_NUMBER=$2
+RELEASE_ID=$3
+WEBHOOK_URL=$4
+
+RELEASE_URL="https://dev.azure.com/dfe-ssp/S118-Teacher-Payments-Service/_releaseProgress?_a=release-pipeline-progress&releaseId=$RELEASE_ID"
+
+MESSAGE=":ship: Build $BUILD_NUMBER is on its way to $ENVIRONMENT_NAME! $RELEASE_URL :rocket:"
+
+curl -X POST -H 'Content-type: application/json' --data "{\"text\":\"$MESSAGE\"}" $WEBHOOK_URL
```

Comment:
Do we have `curl` available?

Reply:
Yarp - the Hosted Ubuntu1604 has in on as standard https://github.com/Microsoft/azure-pipelines-image-generation/blob/master/images/linux/Ubuntu1604-README.md

Reply:
:+1: